integreat_cms/cms/templates/dashboard/dashboard.html

```diff
@@ -17,6 +17,10 @@ <h1 class="heading">{% translate "My Dashboard" %}</h1>
                 {% endif %}
                 <!-- Translation Status Widget: TODO-->
                 <!-- App Size Widget: TODO -->
+                <!-- Page HIX Widget-->
+                {% if request.region.hix_enabled %}
```

Comment:
```suggestion
                {% if TEXTLAB_API_ENABLED and request.region.hix_enabled %}
```
Textlab can also be globally disabled.